packages/db/src/migrate.ts

```diff
@@ -1,7 +1,10 @@
-import { migrate } from "drizzle-orm/node-postgres/migrator";
-import { client, db } from ".";
+// import { migrate } from "drizzle-orm/node-postgres/migrator";
+import { drizzle } from "drizzle-orm/postgres-js";
+import { migrate } from "drizzle-orm/postgres-js/migrator";
+import pg from "pg";
```

Comment:
`pg` import is node-postgres, doesn't that mean the drizzle adapters have to be node-postgres too?

# postgres-js code

![image](https://github.com/icssc/ZotMeal/assets/83149801/1ea72fb3-fa0c-4a03-a533-e434eb7c9963)

# node-postgres code

![image](https://github.com/icssc/ZotMeal/assets/83149801/f089ca3c-e5fe-4794-8862-394ed080453a)

Reply:
Ya we should update one or the other to be consistent. Migration script was following drizzle example for the ci tests run.
Works fine tho